src/containers/StructurePage/resourceComponents/WrongTypeError.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import Tooltip from '@ndla/tooltip';
+import { AlertCircle } from '@ndla/icons/editor';
+import styled from '@emotion/styled';
+import { colors } from '@ndla/core';
+import { ResourceWithNodeConnection } from '../../../modules/nodes/nodeApiTypes';
+import { getIdFromUrn } from '../../../util/taxonomyHelpers';
+
+const StyledWarnIcon = styled(AlertCircle)`
+  height: 24px;
+  width: 24px;
+  fill: ${colors.support.red};
+`;
+
+const getArticleTypeFromId = (id?: string) => {
+  if (id?.startsWith('urn:topic:')) return 'topic-article';
+  else if (id?.startsWith('urn:resource:')) return 'standard';
+  return undefined;
+};
+
+interface Props {
+  resource: ResourceWithNodeConnection;
+  articleType?: string;
+}
+
+const WrongTypeError = ({ resource, articleType }: Props) => {
+  const { t } = useTranslation();
+  const isArticle = resource.contentUri?.startsWith('urn:article');
+  const expectedArticleType = useMemo(() => getArticleTypeFromId(resource.id), [resource]);
+
+  if (!isArticle) return null;
+  if (expectedArticleType === articleType) return null;
```

Comment:
Om !isArticle skal føre til kjapp retur kan du like gjerne gjøre det _før_ du henter ut artikkeltype. 
```suggestion
  if (!resource.contentUri?.startsWith('urn:article')) return null;

  const expectedArticleType = useMemo(() => getArticleTypeFromId(resource.id), [resource]);
  if (expectedArticleType === articleType) return null;
```